ckan/config/deployment.ini_tmpl

```diff
@@ -72,6 +72,8 @@ ckan.auth.user_delete_organizations = true
 ckan.auth.create_user_via_api = false
 ckan.auth.create_user_via_web = true
 ckan.auth.roles_that_cascade_to_sub_groups = admin
+ckan.auth.public_user_details = true
+ckan.auth.public_activity_stream_detail = true
```

Comment:
are these the defaults we want in the ini template?

Reply:
`ckan.auth.public_user_details` is unrelated to this PR but I noticed it was missing so added it for completeness, same value as the default.

`ckan.auth.public_activity_stream_detail` is true to encourage new ckan sites to have an open history. It's different to the default, so that old sites that upgrade have no change unless they consciously set this to true.